### PR TITLE
AN-229: Rename Admin CLI command derive-sponsor-wallet-address

### PR DESCRIPTION
**Pull request by acenolaza**
More details: https://api3dao.atlassian.net/browse/AN-229

### CODE REVIEW

File: packages/admin/README.md [hunk 1]
```diff
@@ -61,12 +61,12 @@ Please, refer to the implementation for more details.
 
 ## Developer commands
 
-### `derive-sponsor-wallet-address`
+### `derive-sponsor-wallet`
 
-Derives the address of the wallet designated by an Airnode for a requester. You need to specify the xpub but if it is not provided then this command will try to fetch it from the AirnodeRrp contract.
+Derives a sponsorWallet designated by an Airnode for a sponsor and returns the address of the wallet. You need to specify the xpub but if it is not provided then this command will try to fetch it from the AirnodeRrp contract.
```
Comment (Siegrift) [inline packages/admin/README.md:66]: I am not a fan of `sponsorWallet` in the README description. Why not just `sponsor wallet`?

**Review by Siegrift — APPROVED** [review]

**Review by Siegrift — COMMENTED** [review]
On the second thought, I am not sure about this change - I think it's more consistent with other commands with the `address` suffix.

The issues says:
```
The command `derive-sponsor-wallet` creates a sponsorWallet and returns its public address.
```
but that is confusing since this command doesn't **create** the wallet, it just derives the wallet address (public key)

**Comment by acenolaza** [comment]
> On the second thought, I am not sure about this change - I think it's more consistent with other commands with the `address` suffix.
> 
> The issues says:
> 
> ```
> The command `derive-sponsor-wallet` creates a sponsorWallet and returns its public address.
> ```
> 
> but that is confusing since this command doesn't **create** the wallet, it just derives the wallet address (public key)

@bbenligiray would be the best person to decide between the 2 options at this point.

**Review by andreogle — APPROVED** [review]

**Comment by bbenligiray** [comment]
Yeah I agree that it should go `derive-sponsor-wallet-address` because it's a bit misleading currently

**Pull request closed without merging**